x-pack/plugins/apm/server/lib/service_group_query_with_overflow.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { QueryDslQueryContainer } from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
+import { kqlQuery } from '@kbn/observability-plugin/server';
+import { SERVICE_NAME } from '../../common/es_fields/apm';
+import { ServiceGroup } from '../../common/service_groups';
+
+export function serviceGroupWithOverflowQuery(
+  serviceGroup?: ServiceGroup | null
+): QueryDslQueryContainer[] {
+  if (serviceGroup) {
+    const serviceGroupQuery = kqlQuery(serviceGroup?.kuery);
+    const otherBucketQuery = kqlQuery(`${SERVICE_NAME} : "_other"`);
```

Comment:
do we need to use `kqlQuery` here? I'd rather write out the ES DSL query.

Reply:
Consistency with everywhere in our code and also it looks neat ?

Reply:
Would this give the same?

```suggestion
    const otherBucketQuery = termsQuery(SERVICE_NAME, '_other');
```